pkg/rc/manager.go

```diff
@@ -0,0 +1,183 @@
+package rc
+
+import (
+	"github.com/square/p2/Godeps/_workspace/src/github.com/Sirupsen/logrus"
+
+	"github.com/square/p2/pkg/kp/rcstore"
+	"github.com/square/p2/pkg/labels"
+	"github.com/square/p2/pkg/logging"
+	"github.com/square/p2/pkg/rc/fields"
+)
+
+// The RCManager is responsible for spawning and reaping replication controllers
+// as they are added to and deleted from Consul. Multiple managers can exist
+// simultaneously, but each one must hold a different Consul session. This
+// ensures that the managers do not instantiate the same replication controller
+// multiple times.
+type RCManager struct {
```

Comment:
I sympathize with that and I am also okay with the name `RCFarm`. thoughts from anyone else? otherwise I'll change it